src/Features/Core/Portable/ChangeSignature/AbstractChangeSignatureService.cs

```diff
@@ -248,23 +230,21 @@ internal ChangeSignatureResult ChangeSignatureWithContext(ChangeSignatureAnalysi
 
 #nullable enable
```

Comment:
Is there anything left in this file that isn't #nullable enabled or isn't easily updated?